[1/20] cd /wrkdirs/usr/ports/devel/kde-dev-utils/work/.build && 
          /usr/local/bin/cmake -DGETTEXT_MSGFMT_EXECUTABLE=/usr/local/bin/msgfmt 
                               -DCOPY_TO=/wrkdirs/usr/ports/devel/kde-dev-utils/work/.build/locale 
                               -DPO_DIR=/wrkdirs/usr/ports/devel/kde-dev-utils/work/kde-dev-utils-18.04.3/po 
                               -P
CMake Error: No script specified for argument -P


Obtained from: https://phabricator.kde.org/D14066

diff --git a/CMakeLists.txt b/CMakeLists.txt
--- CMakeLists.txt
+++ CMakeLists.txt
@@ -10,5 +10,8 @@
 include(KDECompilerSettings NO_POLICY_SCOPE)
 include(ECMQtDeclareLoggingCategory)
 
+# Required for ki18n_install(po) added on tarball generation
+find_package(KF5 REQUIRED COMPONENTS I18n)
+
 add_subdirectory(kpartloader)
 add_subdirectory(kuiviewer)
diff --git a/kpartloader/CMakeLists.txt b/kpartloader/CMakeLists.txt
--- kpartloader/CMakeLists.txt
+++ kpartloader/CMakeLists.txt
@@ -11,7 +11,6 @@
 find_package(KF5 REQUIRED
     COMPONENTS
     CoreAddons
-    I18n
     Parts
     WidgetsAddons
 )
diff --git a/kuiviewer/CMakeLists.txt b/kuiviewer/CMakeLists.txt
--- kuiviewer/CMakeLists.txt
+++ kuiviewer/CMakeLists.txt
@@ -15,7 +15,6 @@
     COMPONENTS
     CoreAddons
     KIO
-    I18n
     Parts
 )
 


